### PR TITLE
Fix the bug that dora worker is blocked by http server

### PR DESCRIPTION
**Pull request by JiamingMai**
Fix the bug that dora worker is blocked by http server

### CODE REVIEW

**Review by yyongycy — APPROVED** [review]
lgtm

File: dora/core/common/src/main/java/alluxio/conf/PropertyKey.java [hunk 1]
```diff
@@ -7728,6 +7728,21 @@ public String toString() {
...
+      booleanBuilder(Name.WORKER_HTTP_SERVER_ENABLED)
+          .setDefaultValue(true)
+          .setDescription("Whether to enable worker HTTP server.")
+          .setConsistencyCheckLevel(ConsistencyCheckLevel.ENFORCE)
+          .setScope(Scope.WORKER)
+          .build();
+
+  public static final PropertyKey WORKER_HTTP_SERVER_PORT =
+      intBuilder(Name.WORKER_HTTP_SERVER_PORT)
+          .setDefaultValue(28080)
```
Comment (yyongycy) [inline dora/core/common/src/main/java/alluxio/conf/PropertyKey.java:7741]: Is this port following some rule? Or just pick a random one?

**Comment by yyongycy** [comment]
alluxio-bot, merge this please

File: dora/core/common/src/main/java/alluxio/conf/PropertyKey.java (hunk 1 above)
Comment (JiamingMai) [reply dora/core/common/src/main/java/alluxio/conf/PropertyKey.java:7741]: Just pick a random one.

**Pull request merged**